src/main/java/com/learning/browsers/DriverModule.java

```diff
@@ -0,0 +1,29 @@
+package com.learning.browsers;
+
+import com.google.inject.Exposed;
+import com.google.inject.PrivateModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import com.learning.configuration.PropertiesReader;
+import org.openqa.selenium.WebDriver;
+
+public class DriverModule extends PrivateModule {
+
+  @Provides
+  @Exposed
+  @Singleton
+  public PropertiesReader providePropertiesReader() {
+    return new PropertiesReader();
+  }
+
+  @Provides
+  @Exposed
+  @Singleton
+  public WebDriver provideWebDriver(PropertiesReader propertiesReader) {
+    return new WebDriverFactory(propertiesReader).getWebDriver();
+  }
+
+  @Override
+  protected void configure() {
```

Comment:
This method can be removed

Reply:
`PrivateModule` implements `Module` interface and `configure()` method is required by contract.